lib/context_adapter_server.js

```diff
@@ -274,6 +319,11 @@ function requestHandler(request, reply) {
 
   if (caComm.isPollingUpdateContext(request)) {
     respond2Request(request, reply);
+  } else if (caComm.getCellIdDescriptors(request).length) {
+    async.parallel([
+      respond2Request.bind(null, request, reply),
+      processCellIdRequest.bind(null, request)
```

Comment:
Not sure what's going to happen once the request has been replied... maybe is not such a good option to make this two calls in parallel.